src/commands/deployWorkspaceProject/deployWorkspaceProject.ts

```diff
@@ -15,28 +15,39 @@ import { localize } from "../../utils/localize";
 import { type DeployWorkspaceProjectResults } from "../api/vscode-azurecontainerapps.api";
 import { browseContainerApp } from "../browseContainerApp";
 import { type DeployWorkspaceProjectContext } from "./DeployWorkspaceProjectContext";
+import { getDefaultContextValues } from "./getDefaultValues/getDefaultContextValues";
 import { getDeployWorkspaceProjectResults } from "./getDeployWorkspaceProjectResults";
 import { deployWorkspaceProjectInternal, type DeployWorkspaceProjectInternalContext } from "./internal/deployWorkspaceProjectInternal";
 
 export async function deployWorkspaceProject(context: IActionContext & Partial<DeployWorkspaceProjectContext>, item?: ContainerAppItem | ManagedEnvironmentItem): Promise<DeployWorkspaceProjectResults> {
+    const subscription: AzureSubscription = await subscriptionExperience(context, ext.rgApiV2.resources.azureResourceTreeDataProvider);
```

Comment:
Any reason for moving this? 

If we have an item, we can get the subscription information from that, so I feel like it should probably be below that check still. Granted, it doesn't look like we were doing that before, but we probably should have been.

Reply:
Oh that's a good point, I'll move it back below.  And we can consider grabbing the subscription if an item is passed in for the V2 iteration